pkg/option/config.go

```diff
@@ -1603,6 +1612,10 @@ type DaemonConfig struct {
 	// IPSec key file for stored keys
 	IPSecKeyFile string
 
+	EnableWireguard   bool
+	WireguardSubnetV4 *net.IPNet
+	WireguardSubnetV6 *net.IPNet
```

Comment:
These should have godocs

Reply:
Also, these should also be put under the `Networking` section for organizing the flags, right?